apps/settings/js/simcard_dialog.js

```diff
@@ -440,17 +476,14 @@ function SimPinDialog(dialog) {
   function show(action, options) {
     options = options || {};
 
+    var dialogPanel = '#' + dialog.id;
     icc = getIccByIndex(options.cardIndex);
-    if (!icc) {
-      return;
-    }
+    _action = initUI(action);
```

Comment:
We should not call to `initUI` if there is no icc because the function requires icc.